src/driver/sap/SapDriver.ts

```diff
@@ -820,20 +805,10 @@ export class SapDriver implements Driver {
      */
     protected loadDependencies(): void {
         try {
-            const client = this.options.driver || PlatformTools.load("hdb-pool")
-            this.client = client
-        } catch (e) {
-            // todo: better error for browser env
-            throw new DriverPackageNotInstalledError("SAP Hana", "hdb-pool")
-        }
-
-        try {
-            if (!this.options.hanaClientDriver) {
```

Comment:
The point of this was that users could use `hdb` instead of `@sap/hana-client`. But I tried to use it a few years ago and it didn't work without patching both `hdb` and and `typeorm`. Also, `hdb` doesn't have built-in pooling anyway, it relied on `hdb-pool`.

So I guess it's fine 🤷🏻